integration_tests_quick_start_guide.rst

```diff
@@ -0,0 +1,646 @@
+
+The basic entity of Ansible integration tests is a ``target``.
+
+The target is an `Ansible role <https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html>`_ stored in the ``tests/integration/targets`` directory of a collection repository file tree.
+
+The target role contains all needed to test a module.
+
+Names of targets contain a module name they test.
+
+Target names that start with ``setup_`` are executed as dependencies before module and plugin targets start execution. We will describe this kind of targets later in detail in the :ref:`Writing tests from scratch<Writing-tests-from-scratch>` section.
+
+To run integration tests, we use the ``ansible-test`` utility shipped with the ``ansible-core`` and ``ansible`` packages. This is described in the :ref:`Run integration tests<Run-integration-tests>` section.
+
+After you finish your integration tests, refert to the `Create a PR quick-start guide <https://github.com/ansible/community-docs/blob/main/create_pr_quick_start_guide.rst>`_ to learn how to submit a pull request.
+
+.. _Prepare-local-environment:
+
+Prepare local environment
+=========================
+
+Before starting working on integration tests, to be able to run them locally, you need to prepare your environment.
+
+To learn how to do it quickly, refer to the `Quick-start development guide <https://github.com/ansible/community-docs/blob/main/create_pr_quick_start_guide.rst#prepare-your-environment>`_.
+
+.. _Determine-if-integration-tests-exists
+
+Determine if integration tests exist
+====================================
+
+Provided that integration tests for a collection exist, they are stored in ``tests/integration/targets`` subdirectories in the collection repository.
+
+If you already have your local environment :ref:`prepared<Prepare-local-environment>`, you can run the following command being in the collection's root directory to list all the available targets:
+
+.. code:: bash
+
+  ansible-test integration --list-targets
+
+If you use ``bash`` and the ``argcomplete`` package is installed via ``pip`` on your system, you can also get a full target list by doing: ``ansible-test integration <tab><tab>``.
+Alternatively, you can check if the ``tests/integration/targets`` contains a corresponding directory named as the module.
+
+For example, the tests for the ``postgresql_user`` module of the ``community.postgresql`` collection are stored in the ``tests/integration/targets/postgresql_user`` directory of the collection's source tree.
+
+If there is no corresponding target there, it means that the module does not have integration tests. In this case, think of adding integration tests for the module. Refer to the :ref:`Writing tests from scratch<Writing-tests-from-scratch>` section for details.
+
+.. _Adding-tests-to-existing-ones:
+
+Adding your tests to existing ones
+==================================
+
+The test tasks are stored in the ``tests/integration/targets/<target_name>/tasks`` directory.
```

Comment:
Do you also want to mention runme.sh based tests? I think it's worth at least mentioning them, so readers of this document aren't completely surprised when they see them.

Reply:
I've no idea what is this:))) I can see this mentioned briefly only once in `ansible/docs/docsite/rst/dev_guide/developing_collections_testing.rst`.
Is it an alternative to `setup_*` targets?
If you know how it works, could you please suggest good explanation for that?:) Would be really interesting to learn the  alternative concept.

Reply:
added, thanks:)